src/components/Banner.js

```diff
@@ -15,56 +15,63 @@ const StyledBanner = styled.section`
   ${sectionMargins()};
 `;
 
-const StyledInner = styled.section`
+const StyledInner = styled.div`
   display: grid;
   gap: 30px;
 
-  ${minBreakpointQuery.small`
-    gap: 40px;
-  `}
-
   ${minBreakpointQuery.medium`
+    gap: 150px;
```

Comment:
150 is quite a lot to have initially, so probably start it at like 40/50 and then increase over 3 or 4 breakpoints